### ISSUE TITLE
Skript removes 'a' when strings starts with 'a'

### ISSUE THREAD

**Issue by TPGamesNL** [SkriptLang/Skript#2012]
### Description
Whenever a string starts with `a` (can be followed by a space), then an expression, it will remove the `a` (and the space if it was there)

### Steps to Reproduce
```
command /bug:
	trigger:
		message "a %sender%"
```
Output: 
![image](https://user-images.githubusercontent.com/29547183/56427654-f4168100-62bc-11e9-9d74-b5a43e97ffdc.png)

### Expected Behavior
It should output `a TPGamesNL`

### Server Information
* **Server version/platform:** Spigot 1.13.2
* **Skript version:** 2.3.6

### Additional Context
It also seems to work with `A` (capital letter), but not with any other characters (as far as I know)

**Comment by FranKusmiruk** [comment]
Duplicate of #367.

TL;DR: it's a feature, just a weird one.
